From 475b20c2d79c69cf4006f0a7920eb093b260d29d Mon Sep 17 00:00:00 2001
From: Robby Cai <R63905@freescale.com>
Date: Thu, 8 Nov 2012 14:45:56 +0800
Subject: ENGR00232893 mx6sl: pxp/v4l2: add dependency on
 CONFIG_FB_MXC_ELCDIF_FB

Without this dependency, we have to manually disable
CONFIG_VIDEO_MXC_PXP_V4L2 when disable CONFIG_FB_MXC_ELCDIF_FB.
Otherwise, a build error shows up. This patch fixed it.

Signed-off-by: Robby Cai <R63905@freescale.com>
---
 drivers/media/video/Kconfig |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/video/Kconfig b/drivers/media/video/Kconfig
index e0b1633..a872632 100644
--- a/drivers/media/video/Kconfig
+++ b/drivers/media/video/Kconfig
@@ -652,6 +652,7 @@ config VIDEO_MXS_PXP
 config VIDEO_MXC_PXP_V4L2
         tristate "MXC PxP V4L2 driver"
         depends on VIDEO_DEV && VIDEO_V4L2 && (ARCH_MX5 || SOC_IMX6SL)
+        select FB_MXC_ELCDIF_FB
         select VIDEOBUF_DMA_CONTIG
         ---help---
           This is a video4linux driver for the Freescale PxP
-- 
1.7.9.5

